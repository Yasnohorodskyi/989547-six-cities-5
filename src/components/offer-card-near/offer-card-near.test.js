import React from "react";
import renderer from "react-test-renderer";

import NearCard from "./offer-card-near";

jest.mock(`../offer-card/offer-card`, () => `OfferCard`);

it(`Should near card render`, () => {
  const tree = renderer
  .create(<NearCard />).toJSON();

  expect(tree).toMatchSnapshot();
});
