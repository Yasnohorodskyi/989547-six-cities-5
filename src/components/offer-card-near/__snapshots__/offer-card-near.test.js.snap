// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should near card render 1`] = `
<OfferCard
  block="near-places"
  className="near-places__card"
  imgHeight="200"
  imgWidth="260"
/>
`;
